src/main/java/frc/robot/commands/RobotLevel.java

```diff
@@ -14,27 +14,18 @@
 import frc.robot.subsystems.Leveling;
 
 /**
- *
+ * its the final countdoooooooooooooooooooooooooooooooooooooooooown
```

Comment:
- [ ] Remove this please. Describe what `RobotLevel` actually does if you want to put something 

```suggestion
 * 
```